### PR TITLE
Add POI tooltips

### PR DESCRIPTION
**Pull request by Dathwada**
This should add the feature for POI tooltips. Still have to test this.

There is a TEST NODE in Azj-Kahet.
Updated the entrance POI for Rrres in Azj-Kahet, this POIs will have the tooltip "Entrance".

**If you want to test this, please reinstall the plugins as I have updated the template.xml and this file is not symlinked.**

- [x] Test it.
- [x] (Maybe add a new Entrance POI class with "Entrance" label as default?)*
- [ ] ~~(Add that you can focus nodes on the minimap)*~~
       I think this is not possible until [HandyNotes ](https://github.com/Nevcairiel/HandyNotes/blob/77816cffe9123a98c2e50faefbfa6b86bce375f5/HandyNotes.lua#L474) allows OnClick
- [ ] (Add that you also focus parent nodes on the world map when focusing the child node)*
- [x] Update existing POIs.
- [x] Get the tooltip working on minimap POIs.
- [x] Remove TestNodes in Azj-Kahet before merging this.

*optional

### CODE REVIEW

**Comment by Dathwada** [comment]
For the Task "Add that you can focus nodes on the minimap"

I think this is not possible until [HandyNotes ](https://github.com/Nevcairiel/HandyNotes/blob/77816cffe9123a98c2e50faefbfa6b86bce375f5/HandyNotes.lua#L474) allows OnClick.

I edited this line from `icon:SetScript("OnClick", nil)` to `icon:SetScript("OnClick", pinsHandler.OnClick)` and it worked plug and play.
Also you can use the context menu on the minimap buttons and everything works.

**Comment by Krovikan** [comment]
`I think this is not possible until [HandyNotes ](https://github.com/Nevcairiel/HandyNotes/blob/77816cffe9123a98c2e50faefbfa6b86bce375f5/HandyNotes.lua#L474)allows OnClick.`

Is not the best make a Issue of Petition Request in https://github.com/Nevcairiel/HandyNotes/issues ?

Seems that Nevcairiel is working again in HandyNotes.

Regards

**Comment by Dathwada** [comment]
~~I think the Lint doesn't allow me to use localizations in core files.~~ NVM

File: core/map.lua [hunk 1]
```diff
@@ -313,6 +317,35 @@ function MinimapPinMixin:UpdateRotation()
...
+            GameTooltip:SetText(ns.RenderLinks(self.label, true))
+
+            -- note
+            if self.note and ns:GetOpt('show_notes') then
+                GameTooltip:AddLine(ns.RenderLinks(self.note), 1, 1, 1, true)
+            end
+
+            GameTooltip:Show()
+        end)
+    end
```
Comment (zarillion) [inline core/map.lua]: This code is indentical for both minimap pins and map pins. Can we move it to a shared function that is called here? Something like this:

```lua
local function RenderPinTooltip(pin)
    ...
end

function MinimapPinMixin:OnMouseEnter()
    if self.label then
        RenderPinTooltip(self)
    end
end

function WorldMapPinMixin:OnMouseEnter()
    if self.label then
        RenderPinTooltip(self)
    end
end
```
```

Comment (Dathwada) [reply core/map.lua]: I moved this.

**Comment by Dathwada** [comment]
@zarillion Would you like to look at this again before merging it? If I get the OK, I will remove the test nodes and then merge it.

**Comment by zarillion** [comment]
The approach makes sense to me. Go ahead and remove the test nodes and lets get this merged!

**Pull request merged**